scripts/pi-hole/js/queries.js

```diff
@@ -282,7 +282,7 @@ function formatInfo(data) {
     var regexLink =
       '<a href="groups/domains?domainid=' +
       data.regex_id +
-      '" target="_blank">regex ID ' +
+      '" target="_blank">domain with ID ' +
```

Comment:
As your screenshot shows, the column name is "Domain/Regex":

![image](https://github.com/pi-hole/web/assets/1385443/c9f7d5ad-6f5b-4f36-9991-7d448a4841ad)

What do you think about using this text?
_(This is just a suggestion)_

```suggestion
      '" target="_blank">entry with ID ' +
```

Reply:
`entry` is good. I tried to avoid `domain/regex` as it is to long, an thought `domain` is more generic. But `entry` is even better.

Reply:
We use `item` in FTL, but this is even more generalized as the code where we are using this handles groups, clients, domains, regex, and lists.

Reply:
So...you mean `entry` on the web interface is fine? Or should it be changed to `item`?

Reply:
I'm fine with both, just figured I mention it